### ISSUE TITLE
[Hold #11111] [$250] Closing on profile path doesn't show the right URL (breaks the URL) - Reported by @aneequeahmad

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#10885]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open web app https://new.expensify.com/
2. Click on Profile.
3. Click on cross on top right to close profile.

## Expected Result:
Url should be r/reportId

## Actual Result:
Url is https://new.expensify.com/settings/profile

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.1.96-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 

https://user-images.githubusercontent.com/43995119/189033143-fa4d0ede-ba25-4598-b5f3-36f276817cad.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @aneequeahmad
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1661278221175169

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by joekaufmanexpensify** [comment]
Hm, I can't reproduce this in staging or production. Closing for now, as it doesn't seem like this is app-wide issue.

**Comment by aneequeahmad** [comment]
@joekaufmanexpensify this issue is still reproducible on production v1.1.98-1. I feel like the navigation functionality of settings page overall needs improvements.

Video attached: 

https://user-images.githubusercontent.com/36359331/189482483-e4494c2c-8dab-4724-93a4-3dcbb3864be0.mov

**Comment by joekaufmanexpensify** [comment]
Hm, yeah. I just tried again now and was able to reproduce now on v1.2.0-8 when navigating to the settings page and then closing.

**Comment by bondydaa** [comment]
this should be fine for a contributor to fix

**Comment by bondydaa** [comment]
whoops sorry Hax didn't mean to unassign you

**Comment by deetergp** [comment]
This should be weekly till it has a proposal right?

**Comment by rushatgabhane** [comment]
cool, thanks for letting us know @parasharrajat

**Comment by michaelhaxhiu** [comment]
@parasharrajat @rushatgabhane @jasperhuangg do you recommend we hold doubling price of this since it's being tracked on #11111 ?

**Comment by michaelhaxhiu** [comment]
Cool, let's hold ⏸️ this one alongside #11111

**Comment by deetergp** [comment]
Still holding. Should we add `[HOLD]` to the title?

**Comment by michaelhaxhiu** [comment]
Making monthly since this is on hold.

**Comment by michaelhaxhiu** [comment]
Held still. Keeping assigned while I'm OOO for 1.5 weeks. If this becomes unblocked please re-assign a new BZ @deetergp 🙏

**Comment by JmillsExpensify** [comment]
Taking this over to manage this and the other issue on hold for #11111

**Comment by JmillsExpensify** [comment]
Still held.

**Comment by JmillsExpensify** [comment]
Still held on the linked issue as well as the navigation re-write.

**Comment by JmillsExpensify** [comment]
Still on hold for navigation. No ETA.

**Comment by JmillsExpensify** [comment]
Same same.

**Comment by deetergp** [comment]
Still in a holding pattern

**Comment by JmillsExpensify** [comment]
Still in a holding pattern.

**Comment by deetergp** [comment]
Still on hold.

**Comment by deetergp** [comment]
MOTS (More of the same)

**Comment by JmillsExpensify** [comment]
Working through the old bugs from the navigation re-write

**Comment by deetergp** [comment]
Still on hold.

**Comment by deetergp** [comment]
Still holding

**Comment by JmillsExpensify** [comment]
Alright, circling back to this issue. It's no longer possible since we've deprecated the X. As a result, @aneequeahmad I've just sent you an offer via Upwork for reporting. Can you accept and we'll get you paid out before closing this issue?

**Comment by JmillsExpensify** [comment]
Great all paid out.
